src/Tools/dotnet-monitor/dotnet-monitor.csproj

```diff
@@ -74,7 +74,10 @@
     </ItemGroup>
     <ItemGroup>
       <!-- Pack the file if it exists. -->
-      <TfmSpecificPackageFile Include="@(AdditionalPackageFile->Exists())" />
+      <TfmSpecificPackageFile Include="@(AdditionalPackageFile-&gt;Exists())" />
+      <TfmSpecificPackageFile Include="$(ArtifactsBinDir)Microsoft.Diagnostics.Monitoring.StartupHook\$(Configuration)\net6.0\Microsoft.Diagnostics.Monitoring.StartupHook.dll">
```

Comment:
Will do this here. I'm also going to add the symbols file and publish the project so that it's in a location that the archive routine used in the pipelines expects.

Reply:
Done